core-field-editors/package.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
one last comment from me, should we maybe add an npm [shrinkwrap](https://docs.npmjs.com/cli/shrinkwrap) so we make sure all users will have the exact same versions